ui/appui-react/src/appui-react/statusfields/message-center/MessageCenterField.scss

```
@@ -0,0 +1,28 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+@import "~@itwin/core-react/lib/cjs/core-react/z-index";
+@import "../../layout/footer/variables.scss";
+
+.uifw-statusFields-messageCenter-popover {
+  width: 305px;
+  min-height: 200px;
+}
+
+.uifw-statusFields-messageCenter-messageCenterField {
+  width: 305px;
+  max-height: 200px;
+  overflow-y: scroll;
```

`scroll` should pretty much never be used. I believe you meant to use [`auto`](https://developer.mozilla.org/en-US/docs/Web/CSS/overflow#auto) here.

You can also add [`scrollbar-gutter`](https://developer.mozilla.org/en-US/docs/Web/CSS/scrollbar-gutter) to prevent layout shift.

```suggestion
  overflow: auto;
  scrollbar-gutter: stable;
```